src/miner.cpp

```diff
@@ -201,8 +201,6 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     pblocktemplate->vchCoinbaseCommitment = GenerateCoinbaseCommitment(*pblock, pindexPrev, chainparams.GetConsensus());
     pblocktemplate->vTxFees[0] = -nFees;
 
-    LogPrintf("CreateNewBlock(): block weight: %u txs: %u fees: %ld sigops %d\n", GetBlockWeight(*pblock), nBlockTx, nFees, nBlockSigOpsCost);
```

Comment:
I am hesitant in removing this log because it is the only indication *(by default)* that the daemon has attempted to mine a block.

However, as stated it quickly fill up `debug.log`.

Reply:
An alternate solution to this issue would be to rotate the `debug.log` automatically to reduce usage. 

There is also an option `-shrinkdebuglog=1` that shrinks the `debug.log` everytime the daemon is started. If it is larger than ~1MB than it is reduced to ~200KB.

Reply:
Optionally maybe just comment out the code so its easily accessible for those who might needed. In my use the log just gets filled up with this making using it for anything really annoying and often resulting in debug.log that is multiple gigabytes.

Reply:
Instead of commenting it out, we can make the log only appear if `debug=1` or `debug=bench` has been set. This is similar behavior to the one below. 

Example:
```cpp
LogPrint(BCLog::BENCH, "CreateNewBlock(): block weight: %u txs: %u fees: %ld sigops %d\n", GetBlockWeight(*pblock), nBlockTx, nFees, nBlockSigOpsCost);
```

Reply:
I think that would be the best course of action, as this should not be usually needed unless debugging.

Reply:
If you make the change with the example above, I'll test it and approve it. It will be merged within 24 hrs to allow any other comments.